### PR TITLE
[bitnami/*] Parametrizing teams for the support workflows

### PR DESCRIPTION
**Pull request by agomezmoron**
### Description of the change

* Removing _triage_ and _support_ teams as fixed variables in the .env file.
* Adding an action to update the BITNAMI_TEAM variable in the .env file based on GH team.
* Changing the assignations by using GH teams instead of an array of users.
* Changing the GH token into the bitnami-bot one.

### Benefits

Managing the triage, support and bitnami teams by managing them on GH.

### Possible drawbacks

None detected.

### CODE REVIEW

File: .github/workflows/sync-teams.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          token: ${{ secrets.BITNAMI_BOT_TOKEN }}
+          fetch-depth: 1
+      - name: Load .env file
+        uses: xom9ikk/dotenv@v1.0.2
+        with:
+          path: .github/workflows/
+      - name: Updating members of the Bitnami team
+        run: |
+          TEAM_MEMBERS=$(curl --request GET \
+          --url https://api.github.com/repos/orgs/bitnami/teams/developers/members \
```
Comment (fmulero) [inline .github/workflows/sync-teams.yml:25]: Shouldn't it be?
```suggestion
          --url https://api.github.com/orgs/bitnami/teams/developers/members \
```

File: .github/workflows/sync-teams.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+          --header 'content-type: application/json' \
+          | jq 'sort_by(.login)|map(.login)|join(",")') \
```
Comment (fmulero) [inline .github/workflows/sync-teams.yml:28]: I think last `\` is not needed
```suggestion
          | jq 'sort_by(.login)|map(.login)|join(",")')
```

File: .github/workflows/sync-teams.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          TEAM_MEMBERS='['${TEAM_MEMBERS//','/'","'}']'
+          if [ $TEAM_MEMBERS != $BITNAMI_TEAM ]; then
+            echo "Replacing $BITNAMI_TEAM for $TEAM_MEMBERS"
+            cp .github/workflows/.env .github/workflows/.env-copy
+            sed '1d' .github/workflows/.env-copy > .github/workflows/.env
+            echo "BITNAMI_TEAM='$TEAM_MEMBERS'" | cat - .github/workflows/.env > .github/workflows/.env-copy && mv .github/workflows/.env-copy .github/workflows/.env
```
Comment (fmulero) [inline .github/workflows/sync-teams.yml:34]: Not fully tested but I think you can use this:

```suggestion
            sed -i "s|BITNAMI_TEAM=.*$|BITNAMI_TEAM='${TEAM_MEMBERS}'|g" .github/workflows/.env
```

File: .github/workflows/triage.yml [hunk 4]
```diff
@@ -32,34 +32,27 @@ jobs:
...
-          TRIAGE_TEAM_STRING=${TRIAGE_TEAM_STRING/]/}
-          TRIAGE_TEAM_STRING=${TRIAGE_TEAM_STRING//'"'/}
-          # creating env variable "on the fly"
-          echo "TRIAGE_TEAM_STRING=$TRIAGE_TEAM_STRING" >> $GITHUB_ENV
       - name: Assign to a person to work on it
         uses: pozil/auto-assign-issue@v1.7.3
         with:
           numOfAssignee: 1
-          assignees: ${{ env.TRIAGE_TEAM_STRING }}
+          #assignees: ${{ env.TRIAGE_TEAM_STRING }}
```
Comment (fmulero) [inline .github/workflows/triage.yml:39]: Could we remove it?

**Review by fmulero — CHANGES_REQUESTED** [review]
Hi @agomezmoron I left some comments

File: .github/workflows/triage.yml (hunk 4 above)
Comment (agomezmoron) [reply .github/workflows/triage.yml:39]: Ok, I left it as #teams was. But it's true, let's remove it.

File: .github/workflows/sync-teams.yml (hunk 3 above)
Comment (agomezmoron) [reply .github/workflows/sync-teams.yml:34]: I think I tested it and failed. Let me try again

Comment (agomezmoron) [reply .github/workflows/sync-teams.yml:34]: You were fine, I'll update the workflow with that suggestion. Thanks!

**Review by fmulero — APPROVED** [review]
LGTM
